src/main/java/com/minecolonies/core/colony/colonyEvents/raidEvents/AbstractShipRaidEvent.java

```diff
@@ -356,6 +367,19 @@ public void onEntityDeath(final LivingEntity entity)
             status = EventStatus.WAITING;
             MessageUtils.format(ALL_PIRATES_KILLED_MESSAGE, colony.getName()).sendTo(colony).forManagers();
         }
+        raiderKillCount++;
+
+        if (!spawners.isEmpty() && raiderKillCount > maxRaiderCount/maxSpawners)
```

Comment:
that'll destroy one spawner for each dead entity after reaching the threshold, since the threshold doesnt change